client/src/pages/NoMatch/index.jsx

```diff
@@ -1,21 +1,38 @@
 import React from 'react';
 import Nav from '../../components/Nav';
 import noParking from '../../images/NoParking.png';
+import bang from './gifs/bang.gif';
+import bounce from './gifs/bounce.gif';
+import crash from './gifs/crash.gif';
+import ouch from './gifs/ouch.gif';
+import whoops from './gifs/whoops.gif';
 
 import './style.scss';
 
 const NoMatch = () => {
-    window.setTimeout( function(){
-        window.location = "/";
-    }, 4000 );
+    // window.setTimeout(function () {
+    //     window.location = "/";
+    // }, 10000);
```

Comment:
so we don't need the auto-redirect anymore or will we add it back later?

Reply:
it'll be back in the next PR. forgot to uncomment when i pushed